### ISSUE TITLE
INTERNAL_ERROR

### ISSUE THREAD

**Issue by bbenjbb** [F5Networks/f5-google-gdm-templates#10]
Hello,
With "HA Cluster (Active/Standby): Production Stack with PAYG Licensing" deployment
All prequisite ok when I want to deply I have this error message without any more verbose logs :

bbr-f5ha-payg1 has resource warnings
bigip1-bbr-f5ha-payg1: {"ResourceType":"compute.v1.instance","ResourceErrorCode":"INTERNAL_ERROR","ResourceErrorMessage":"Code: '-4207409696167612610'"}

storage bucket and firewall was created but not instances.

in attachment YAML file and screenshot of my network configuration.
[f5-deployment-template.txt](https://github.com/F5Networks/f5-google-gdm-templates/files/2438229/f5-deployment-template.txt)
![screenshot at oct 02 16-15-07](https://user-images.githubusercontent.com/24722803/46354373-bc1c5900-c65e-11e8-92be-74ce3a3a699d.png)

Route to internet from management and external network are present and firewall rules too.

Please help !

**Comment by crosbygw** [comment]
When using deployment-manager, the service account tied to your deployment-manager service is being used to create all resources noted in template. Verify your deployment manager service account has the correct rights to create images into the project you are adding resources. https://cloud.google.com/deployment-manager/docs/access-control
The error noted indicates to me the service account does not have permission to build compute resources.

In addition, for the HA to work, you need  to provide a service account in your configuration file that has the minimum roles:
- Storage Object Creator
- Compute image admin

Alternatively you can leave the service account property blank in your configuration file and your projects default compute service account will be used. It also must have at a minimum the roles noted above. Unless it has been altered, the default compute service account should have editor role which grants access to all compute resources.

I will add a task to have documentation updated to reflect these requirements.

**Comment by bbenjbb** [comment]
ok we found solution to our issue.

This is not a problem with my account (because my account is project owner) but a problem with Alias IP.

comment in the Yaml file :
`   ### Enter alias IP address(es) to be used for application traffic, including CIDR suffix. This address must belong to the subnet noted above in key 'subnet1'.  A list of alias IPs can be provided, seperated by a semi-colon: IE 10.x.x.16/28;10.x.x.32/28.
`

This is not an IP address but a Network address.
In my first Yaml I filled this variable with 
`aliasIp: '10.10.2.17/28'`

With
`aliasIp: '10.10.2.16/28'`

It's ok

Perhaps it will be necessary to check this variable or modify comment.

Thanks
